content/blog/2021/08/2021-02-08.adoc

```diff
@@ -0,0 +1,167 @@
+---
+:layout: post
+:title: "CloudEvents Plugin for Jenkins: Interoperability between Jenkins and CI/CD Tools"
+:tags:
+- gsoc
+- gsoc2021
+- cloudevents
+- interoperability
+- cloud-native
+authors:
+- ShrutiC-git
+description: >
+  This is an introductory blog to the new CloudEvents plugin for Jenkins, developed as a GSoC'21 project. In this blog, we will be looking at how the integration between Jenkins and CloudEvents makes working across a suite of CI/CD tools easier. CloudEvents is the industry-adopted specification for describing events, and with the release of this plugin, Jenkins is now CloudEvents-compliant.
+opengraph:
+  image:/images/post-images/2021-08-cloudevents-plugin/jcdevents.JPG
```

Comment:
I didn't realize that. Thanks for suggesting the edit.

Reply:
Yeah, that is because the front-matter (the lines between the `---` markers at the top) use a different parser than the blog post content. I too was confused by that!